indigo_social/templates/indigo_social/user_popup.html

```diff
@@ -5,7 +5,7 @@
 
   <div class="media mb-2">
     {% user_profile_photo_thumbnail user height=40 width=40 %}
-    <div class="media-body ml-2">
+    <div class="media-body ms-2">
```

Comment:
BS 4's "media object" component is gone, you'll need to adjust this.

Reply:
I can't seem to find where this user_popup template is used within the application. I see a path `/contributors/name/popup` but it renders some isolated html, how can I properly view this? where is it used?

Reply:
hover over a username like in an assigned task view

![image](https://github.com/laws-africa/indigo/assets/4178542/d91e2a4c-7c18-4226-9c67-cbec4184a824)

Reply:
or in https://edit.laws.africa/contributors/

Reply:
fixed: 
![image](https://github.com/laws-africa/indigo/assets/25079238/203d255f-11e5-4b2c-a27d-06f7bf21069a)